R/exclusions.R

```diff
@@ -0,0 +1,99 @@
+#' Convert case counts in matching rows to NA
+#'
+#' NAs are skipped in model fitting by EpiNow2, so matched rows are excluded
+#' from the model likelihood.
```

Comment:
I might suggest using the description of a function to talk about what the function does primarily, before going on to talk about the circumstances in which one might use it. In other words, this function is quite separate from `EpiNow2` and perhaps you'd want to NA some entries of the columns in other circumstances. But I do get why you've written it this way. So my suggest here would be to write another sentence saying what the function does, before the sentence about the implications that has for passing into `EpiNow2`.

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/25/commits/a10afb6744f86d2ac9fb663de63012c6559b9e0c